protoc

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+if [ `uname -s` = "Darwin" ]; then
+  os=osx
+else
+  os=linux
+fi
+arch=`uname -m`
+
+protocbin=.protoc
+protocversion=3.0.0
+protocurl=https://github.com/google/protobuf/releases/download/v$protocversion/protoc-$protocversion-$os-$arch.zip
```

Comment:
prefer ${protocversion} to make it clearer.  Also we should probably quote this Just In Case.